src/Services/Match/Match.Application/UseCases/ProfileUseCases/Commands/Create/CreateProfileHandler.cs

```diff
@@ -0,0 +1,27 @@
+using AutoMapper;
+using Match.Application.DTOs.Profile.Response;
+using Match.Application.Exceptions;
+using Match.Domain.Repositories;
+using MediatR;
+using Profile = Match.Domain.Models.Profile;
+
+namespace Match.Application.UseCases.ProfileUseCases.Commands.Create;
+
+public class CreateProfileHandler(IUnitOfWork _unitOfWork, IMapper _mapper) : IRequestHandler<CreateProfileCommand, ProfileResponseDto>
+{
+    public async Task<ProfileResponseDto> Handle(CreateProfileCommand request, CancellationToken cancellationToken)
+    {
+        var profileRepository = _unitOfWork.Profiles;
```

Comment:
Why are you creating a profileRepository? You can explicitly call without creating a variable